docs/advanced/authentication.md

```diff
@@ -346,13 +346,13 @@ To do this, we'll need to add the `mapExchange` to the exchanges array, _before_
 The order is very important here:
 
 ```js
-import { createClient, dedupExchange, cacheExchange, fetchExchange, errorExchange } from 'urql';
+import { createClient, debugExchange, cacheExchange, fetchExchange, errorExchange } from 'urql';
 import { authExchange } from '@urql/exchange-auth';
 
 const client = createClient({
   url: 'http://localhost:3000/graphql',
   exchanges: [
-    dedupExchange,
+    debugExchange,
```

Comment:
This does not seem quite right 😅 just removing this and replacing the error with the mapExchange seems more correct

Reply:
Sorry. 
Added appExchange. 
Could you check fixed doc?

fixed: https://github.com/urql-graphql/urql/pull/3533/commits/ce534e247cb677b8c1004d5267acee4e2ddeb98d